src/L.Control.Locate.js

```diff
@@ -280,6 +280,7 @@ You can find the project at: https://github.com/domoritz/leaflet-locatecontrol
             createButtonCallback: function (container, options) {
                 var link = L.DomUtil.create('a', 'leaflet-bar-part leaflet-bar-part-single', container);
                 link.title = options.strings.title;
+                link.tabIndex=0;
```

Comment:
So if we are 0, how do we make sure that other buttons come before in the tab order?

Reply:
tabindex="0" means that the element should be focusable in sequential keyboard navigation, after any positive tabindex values and its order is defined by the document's source order.
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

But it should be a char not a number, I'll make that change.

Reply:
Removed